docker-compose.base.yml

```diff
@@ -24,7 +24,7 @@ services:
     command: mongod --port 27017
     logging:
       driver: none
-    image: mongo:7.0.2
+    image: mongo:8.0.5
```

Comment:
Can this be downgraded to the latest available version of v7 to keep this within the scope of patching? While I'm not opposed to major version upgrades, I believe it would be prudent to involve some more rigorous testing - something that I unfortunately lack the bandwidth for at the moment